tests/test_config_gsobject.py

```diff
@@ -45,36 +47,36 @@ def test_gaussian():
                    'dilate' : 3, 'ellip' : galsim.Shear(e1=0.3),
                    'rotate' : 12 * galsim.degrees,
                    'magnify' : 1.03, 'shear' : galsim.Shear(g1=0.03, g2=-0.05),
-                   'shift' : { 'type' : 'XY', 'x' : 0.7, 'y' : -1.2 }
+                   'shift' : { 'type' : 'XY', 'x' : 0.7, 'y' :-1.2 }
```

Comment:
Why??  

It looks to me like you used some robot style formatter on this file.  Please don't.  As you can see, it does stupid things like remove the space between the colon and the number if the number starts with a minus sign.  This makes this less legible, not more.

Please revert all changes to this file other than the functions you added.